app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,58 @@
+
+
+<section id="index_container">
+  <h2 id="to_do"> # TODO</h2>
+  <ul class="to_do_list">
+    <% counter = 0 %>
+    <% @tasks.each do |task| %>
+    <% counter += 1 %>
+    <li>
+      <%= counter %>)
+      <span <%= task.status ? "class=strikethrough"  : "class=normal" %>>
+
+          <%= link_to(task.name, task_path(task.id), alt: "link to task name", class: "task_link") %>
+      </span>
+
+      <span class="sublist_and_box">
+        <ul class="bullet_list">
+          <li >
+            <form>
```

Comment:
Not sure if you'd want to use a <form> tag here, forms are usually for sending and submitting form information